src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1723,17 +1723,14 @@ public String getQuestionText(String feedbackQuestionId) {
                 recipientTeam = getNameForEmail(response.recipient);
             }
 
-            if (!sortedMap.containsKey(recipientTeam)) {
-                sortedMap.put(recipientTeam,
+            sortedMap.putIfAbsent(recipientTeam,
                         new LinkedHashMap<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>());
-            }
+
             Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responsesForOneRecipient =
                                             sortedMap.get(recipientTeam);
```

Comment:
The two statements above can be conveniently combined to:
```java
Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responsesForOneRecipient =
        sortedMap.computeIfAbsent(recipientTeam, new LinkedHashMap<>());
```

Reply:
ok got it! and for the example, do u mean something like this? 

`Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responsesForOneRecipient =
                    sortedMap.computeIfAbsent(recipientTeam,
                            (String key) -> new LinkedHashMap<>());`

Reply:
@wenmogu you're right, your comment came just slightly before I realized the mistake in my example.

Reply:
ok thank u for the suggestion!

Reply:
@wkurniawan07 

Why not `computeIfAbsent` since the value is key-independent.

```java
Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>> responsesForOneRecipient =
        sortedMap.computeIfAbsent(recipientTeam, k -> new LinkedHashMap<>());
```

Reply:
I mean `putIfAbsent` updated :P

Reply:
I'm actually curious about this as well, I am not sure why `Map#putIfAbsent` returns the previous value but using `Map#computeIfAbsent` seems to muddle the readability but is the shorter solution.

Reply:
@LiHaoTan can't argue much if we're not the language developer, can we?